doc/keepalived.conf.SYNOPSIS

```diff
@@ -706,7 +707,8 @@ virtual_server group <STRING>      {  # VS group declaration
                 status_code <INTEGER>
                 virtualhost <STRING>
             }
-            ...
+
+            nb_get_retry # ?
```

Comment:
This keyword is deprecated and so should not be documented. The supported keyword is `retry`.